src/genie/libs/parser/iosxe/show_cable_tdr_interface.py

```diff
@@ -0,0 +1,122 @@
+'''show_cable_tdr_interface.py
```

Comment:
please move this into a file called `show_cable.py`

Reply:
@danielgraziano  Stupid question.  I'm I just renaming the file from show_cable_tdr_interface.py to show_cable.py (moving the code)?  Or is it just moving that one line to show_cable.py

Reply:
You'd want to rename the whole file. Calling it show_cable.py leaves it open to having other `show cable` commands added to it in the future.